spiceaidocs/docs/features/cdc/index.md

```diff
@@ -51,9 +51,10 @@ datasets:
       debezium_transport: kafka
       debezium_message_format: json
       kafka_bootstrap_servers: localhost:19092
+      kafka_security_protocol: PLAINTEXT
```

Comment:
Any quickstarts to change too?

Reply:
We don't have any quickstarts for debezium, only a sample